src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```
each ColorSupplier should have it's random instance, so it shouldn't be static